File: 1051/main.cpp
class Solution {
public:
    int heightChecker(vector<int>& heights) {
        int n = heights.size();
        int rightPosition[n];
        
        for(int i = 0; i < n; i++) {
            rightPosition[i] = heights[i];
        }
        
        sort(rightPosition, rightPosition + n);
        
        int result = 0;
        for(int i = 0; i < n; i++) {
            if(heights[i] != rightPosition[i]) {
                result++;
            }
        }
        
        return result;
    }
};

File: 1051/main.py
class Solution:
    def heightChecker(self, heights: List[int]) -> int:
        n = len(heights)
        rightPosition = [0] * n
        for i in range(n):
            rightPosition[i] = heights[i]
            
        rightPosition.sort()
        
        result = 0
        for i in range(n):
            if rightPosition[i] != heights[i]:
                result += 1
                
        return result
